{{ cookiecutter.project_shortname }}/setup.cfg

```diff
@@ -59,6 +66,17 @@ add_ignore = D401
 [pycodestyle]
 exclude = docs/conf.py
 
+[isort]
+profile=black
+
+[check-manifest]
+ignore =
+    *-requirements.txt
+
+[tool:pytest]
+addopts = --black --isort --pydocstyle --pycodestyle --doctest-glob="*.rst" --doctest-modules --cov={{ cookiecutter.package_name }} --cov-report=term-missing
```

Comment:
I saw mixed results in repos regarding `pycodestyle`. We should probably review linting and what kind of tools we want to use/enforce

Reply:
for black managed packages the `--pycodestyle` has been removed. If i remember correctly there were problems with black and pycodestyle. Further in my opinion pycodestyle is useless if we have an opinionated code formatter

Reply:
Thanks for chipping-in, I couldn't figure out if that was a pattern or random :)

In that case, I'm removing pycodestyle from the pytest flags and any other config.

I was also wondering if we could replace `flake8`, `pycodestyle`, etc. with [`ruff`](https://github.com/charliermarsh/ruff), which is the new kid in the Python linters block:

- written in Rust for performance (10-100x compared to the above tools)
- compatible with `black` [out of the box](https://github.com/charliermarsh/ruff#is-ruff-compatible-with-black)
- widely adopted but big projects

Something for much later though :)

Reply:
i commented on the [inveniosoftware/pytest-invenio#93](https://github.com/inveniosoftware/pytest-invenio/issues/93) about the idea to use `ruff` instead of `pycodestyle` and `flake8`